src/main/java/com/theelixrlabs/healthcare/constants/EndpointConstants.java

```diff
@@ -0,0 +1,9 @@
+package com.theelixrlabs.healthcare.constants;
+
+/**
+ * Contains endpoint constants used throughout the healthcare application.
+ */
+public class EndpointConstants {
```

Comment:
change class name to ApiPathsConstant
Also please add all other related Apis to this class

Reply:
changed the name to ApiPathsConstant  and added all the API's to the class ApiPathsConstant